src/main/java/DeadlineTask.java

```diff
@@ -0,0 +1,44 @@
+public class DeadlineTask extends Task{
+    protected static String deadline;
+
+    public DeadlineTask(boolean status, String description) throws SalmonMissingArgument{
+        this.isDone = status;
+        this.description = setDeadline(description);
+        this.type = "D";
+    }
+
+    /**
+     * setDeadline parse argument of inputted query,
+     * get the deadline,
+     * assign it to the object
```

Comment:
The comment should be meaningful for the reader. Try not to write comments which are private notes to yourself.